index.css

```diff
@@ -12,45 +17,76 @@ h1 {
   color: black;
   text-align: center;
   background-color: lightgoldenrodyellow;
-  padding: 30px;
+  padding: 2em;
   margin: 0;
-  font-size: 20px;
 }
 
+
 .input-container {
-  margin-top: 20px;
+  margin-top: 1.25em;
+  width: auto;
 }
 
 .input-box {
-  height: 40px;
+  height: auto;
   display: flex;
+  flex-direction: column;
 }
 
 
 .input-tag {
-  height: 34px;
-  width: 80%;
-  font-size: 18px;
+  height: 3em;
+  width: auto;
+  font-size: 0.75em;
   
 }
 .enter-btn {
-  height: 40px;
-  width: 20%;
-  margin-left: 20px;
+  height: 2.5em;
+  width: 100%;
+  margin-top: 1em;
+  margin-left: 0;
 }
 
 .label-tag {
   color: olive;
-  font-size: 16px;
+  font-size: 1.75em;
+  text-align: center;
+  margin: 2%;
 }
 
+
 .image {
```

Comment:
This class name .image and .img will make us confused.
.image 
The .image class is responsible for the CSS around the .img.
How about `.img-around` ?
If you have a good name that's OK.